usr/src/uts/common/fs/vnode.c

```diff
@@ -852,6 +852,31 @@ vn_rele(vnode_t *vp)
 	mutex_exit(&vp->v_lock);
 }
 
+void
+vn_phantom_rele(vnode_t *vp)
+{
+	VERIFY(vp->v_count > 0);
+
+	mutex_enter(&vp->v_lock);
+	VERIFY3U(vp->v_count, >=, vp->v_phantom_count);
+	vp->v_phantom_count--;
+	DTRACE_PROBE1(vn__phantom_rele, vnode_t *, vp);
+	if (vp->v_count == 1) {
+		mutex_exit(&vp->v_lock);
+		VOP_INACTIVE(vp, CRED(), NULL);
+		return;
+	}
+	VN_RELE_LOCKED(vp);
+	mutex_exit(&vp->v_lock);
+}
+
+uint_t
```

Comment:
Comment explaining why we need this, please? I know the vnode.h really explains it, but someone passing by here might think, "WTF?"

Reply:
Done.